src/openvic-simulation/dataloader/NodeTools.cpp

```diff
@@ -411,6 +411,17 @@ node_callback_t NodeTools::name_list_callback(callback_t<std::vector<std::string
 	};
 }
 
+std::ostream& OpenVic::operator<<(std::ostream& stream, name_list_t const& name_list) {
```

Comment:
Why?
std::vector<std::string> makes more sense imo

Reply:
I added the type alias to make it clear what the string vector is expected to represent in this operator overload, and to make it clear that only this type has an overloaded print operator, instead of there being more general vector printing operators. Tbh this operator isn't actually needed, I added it because `map_callback` previously also printed the value when the key was duplicate, but I removed that because the requirement that values had to be printable became too restrictive.